.gitignore

```diff
@@ -1,6 +1,9 @@
 # Pycharm
 /.idea
 
+# Default project slug path
+**/MyNewProject
```

Comment:
I don't understand this addition. Why do we ignore MyNewProject?

Reply:
As discussed, I am adding few lines to ignore development files and paths. Thanks, --Miguel